src/main/scala/beam/utils/data/synthpop/IndustryAssigner.scala

```diff
@@ -55,7 +140,49 @@ object IndustryAssigner {
           ((o, d), xs.map(_._2).sum)
       }
       .sortBy(x => -x._2)
-    println(s"homeGeoIdToWorkGeoIdWithCounts ${homeGeoIdToWorkGeoIdWithCounts.size}")
+    homeGeoIdToWorkGeoIdWithCounts
+  }
+
+  private def readFromCsv(path: String): Seq[((String, String), Int)] = {
```

Comment:
Shouldn't we use the csv utility we already have.

Reply:
It is used on this line https://github.com/LBNL-UCB-STI/beam/blob/09ddf870347991388dc2234c9321bafeb5a43705/src/main/scala/beam/utils/data/synthpop/IndustryAssigner.scala#L150